src/sentry/api/endpoints/organization_events_stats.py

```diff
@@ -20,17 +21,27 @@ def get(self, request, organization):
         if not features.has("organizations:discover-basic", organization, actor=request.user):
             return self.get_v1_results(request, organization)
 
+        try:
+            limit = int(request.GET.get("topEvents", 0))
+        except ValueError:
+            return Response({"detail": "topEvents must be an integer"}, status=400)
+        if limit > MAX_TOP_EVENTS:
+            return Response(
+                {"detail": "Can only get up to {} top events".format(MAX_TOP_EVENTS)}, status=400
+            )
+        elif limit <= 0:
```

Comment:
Won't this result in an error in all requests that don't set topEvents as you default the value to 0?

Reply:
👀 woops, yup. Will fix :+1: